### PR TITLE
avm1: Correct `Add`, `Equals`, `Less` and `GetProperty` actions

### PR DESCRIPTION
**Pull request by Toad06**
Follow-up for #6678. This now completely removes `fn into_number_v1` as suggested by Herschel - the same rules indeed apply to these actions.

Here are [the files](https://github.com/ruffle-rs/ruffle/files/8468277/files.zip) I used while testing (they are a bit rough and require JPEXS to change the SWF version, sorry).

### CODE REVIEW

**Comment by Herschel** [comment]
Could you add these SWFs as test cases? I think it's okay if they are created/modified in JPEXS. Thank you!

File: core/src/avm1/activation.rs [hunk 1]
```diff
@@ -575,9 +575,9 @@ impl<'a, 'gc, 'gc_context> Activation<'a, 'gc, 'gc_context> {
     }
 
     fn action_add(&mut self) -> Result<FrameControl<'gc>, Error<'gc>> {
-        let a = self.context.avm1.pop();
-        let b = self.context.avm1.pop();
-        let result = b.into_number_v1() + a.into_number_v1();
+        let a = self.context.avm1.pop().coerce_to_f64(self)?;
```
Comment (relrelb) [inline core/src/avm1/activation.rs:578]: `b` should be coerced before `a`:
```as
var a = { valueOf: function() { trace("a"); } };
var b = { valueOf: function() { trace("b"); } };
trace(a + b); // traces "b", then "a"
```
This is also true for at least `Add2`, and probably others, which in Ruffle the order is wrong.

Comment (Toad06) [reply core/src/avm1/activation.rs]: Good catch, thank you! I fixed `Add`, as well as `Less` - `Equals` has a different behavior. I can do additional testings in the next days to fix `Add2` and check the other actions, unless you already have plans for this.

**Review by relrelb — APPROVED** [review]

**Comment by Herschel** [comment]
Thank you!

**Pull request merged**